Library/Homebrew/formula_installer.rb

```diff
@@ -1139,14 +1139,21 @@ def pour
       )
     end
 
-    tab.tap = formula.tap
+    # fill in missing/outdated parts of the tab
+    # keep in sync with Tab#to_bottle_json
+    tab.used_options = []
+    tab.unused_options = []
+    tab.built_as_bottle = true
     tab.poured_from_bottle = true
-    tab.time = Time.now.to_i
-    tab.head = HOMEBREW_REPOSITORY.git_head
-    tab.source["path"] = formula.specified_path.to_s
     tab.installed_as_dependency = installed_as_dependency?
     tab.installed_on_request = installed_on_request?
+    tab.time = Time.now.to_i
+    tab.compiler = DevelopmentTools.default_compiler
```

Comment:
Not sure I quite get why we're overwriting this. Don't we want the compiler used at build time?

Reply:
@Bo98 we're not overwriting it as it won't be set. My assumption was the default compiler was always the compiler for bottles but I realise there's cases with `fails_with` etc. where that won't be the case. Will fix.